shared/build.gradle.kts

```diff
@@ -69,11 +69,11 @@ kotlin {
     }
 }
 android {
-    compileSdkVersion(29)
```

Comment:
I've left the shared framework's gradle file alone for now since it's a bit different from the others, but I did bump the version to the latest